package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "lint": "eslint **/*.js",
     "test": "jest",
```

Comment:
What happens if user run this nm script? Doesn't it should have combined default-tests and serial-tests npm scripts?

Reply:
It will run all existing projects default-tests and serial-tests.